src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGardenRecipeTypes.java

```diff
@@ -12,8 +12,8 @@ public final class ExoticGardenRecipeTypes {
     private ExoticGardenRecipeTypes() {}
 
     public static final RecipeType KITCHEN = new RecipeType(new NamespacedKey(ExoticGarden.instance, "kitchen"), new SlimefunItemStack("KITCHEN", Material.CAULDRON, "&eKitchen"), "", "&rThis item must be made", "&rin a Kitchen");
-    public static final RecipeType BREAKING_GRASS = new RecipeType(new NamespacedKey(ExoticGarden.instance, "breaking_grass"), new CustomItemStack(Material.GRASS, "&7Breaking Grass"));
-    public static final RecipeType HARVEST_TREE = new RecipeType(new NamespacedKey(ExoticGarden.instance, "harvest_tree"), new CustomItemStack(Material.OAK_LEAVES, "&aHarvesting a Tree", "", "&rYou can obtain this Item by", "&rharvesting the shown Tree"));
-    public static final RecipeType HARVEST_BUSH = new RecipeType(new NamespacedKey(ExoticGarden.instance, "harvest_bush"), new CustomItemStack(Material.OAK_LEAVES, "&aHarvesting a Bush", "", "&rYou can obtain this Item by", "&rharvesting the shown Bush"));
+    public static final RecipeType BREAKING_GRASS = new RecipeType(new NamespacedKey(ExoticGarden.instance, "breaking_grass"), CustomItemStack.create(Material.GRASS_BLOCK, "&7Breaking Grass"));
```

Comment:
`GRASS` was renamed to `SHORT_GRASS` instead of `GRASS_BLOCK` in 1.20.3. also simply using new enum name breaks backward compatibility.